src/test/java/com/jcabi/github/mock/MkIssueTest.java

```diff
@@ -253,6 +255,53 @@ public void assignsUser() throws Exception {
     }
 
     /**
+     * MkIssue can create a closed event when closing an issue.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void closingGeneratesEvent() throws Exception {
+        final Issue.Smart issue = new Issue.Smart(this.issue());
+        issue.close();
+        MatcherAssert.assertThat(
+            issue.events(),
+            Matchers.<Event>iterableWithSize(1)
+        );
+        final Event.Smart closed = new Event.Smart(
+            issue.events().iterator().next()
+        );
+        MatcherAssert.assertThat(
+            closed.type(),
+            Matchers.equalTo(Event.CLOSED)
+        );
+    }
+
+    /**
+     * MkIssue can create a reopened event when closing an issue.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void reopeningGeneratesEvent() throws Exception {
```

Comment:
@cvrebert let's rename the test to something like `createsReopeningEvent`, the name should start with a verb.